apps/els_lsp/test/els_hover_SUITE.erl

```diff
@@ -78,12 +78,21 @@ init_per_testcase(TestCase, Config) ->
 
 -spec end_per_testcase(atom(), config()) -> ok.
 end_per_testcase(TestCase, Config) ->
+    lists:foreach(
```

Comment:
The fact that meck registers a module with the `_meck` suffix is an implementation detail that we should not rely on. We should only unload `els_docs` for those testcases where the module is mocked.

Reply:
The correct way to do "unmock if mocked" is to use the `meck` API instead. `meck:mocked` returns a list of mocked modules.

Reply:
Ah nice to know 👍